docs/source/udf/profiling.rst

```diff
@@ -0,0 +1,180 @@
+.. _`udf profiling`:
+
+.. testsetup:: *
+
+    from libertem.api import Context
+
+
+Profiling UDFs
+==============
+
+To bring your UDF to production, it may be necessary to look into the performance and
+efficiency of your function. There may be some low-hanging fruit to optimize away, but
+they only really become visible once you profile your code. Just looking at code and guessing
+which parts are expensive can be misleading and cause you to spend time optimizing the wrong part
+of your code. Always measure!
+
+Profiling means instrumenting the execution of a program and finding out which parts
+of your code use how much of a given resource (for example, CPU time, or memory). 
+
+Prerequisite: inline executor
```

Comment:
Inline Executor, capitalization? Not sure on this one!

Reply:
I'll use the full name here